### ISSUE TITLE
Question about recipe of Librispeech 100h

### ISSUE THREAD

**Issue by asadolahzade** [espnet/espnet#4069]
Hi,
I have two questions about the recipe for LS-100. Could you please answer them.
1- nbpe for the “LS-960 recipe” is 5000 [link](https://github.com/espnet/espnet/blob/master/egs2/librispeech/asr1/run.sh#L19). In LS-100 recipe in ESPnet1, it is reduced to 300  but the LS-100 recipe in ESPnet2 [link](https://github.com/espnet/espnet/blob/master/egs2/librispeech_100/asr1/run.sh#L24) keeps it the same (5000). Should we reduce nbpe when reducing the size of training data or no need? Can it affect the result?

2- In [link](https://github.com/espnet/espnet/blob/master/egs2/librispeech_100/asr1/conf/tuning/train_asr_conformer_lr2e-3_warmup15k_amp_nondeterministic.yaml#L81) you use variable maximum width for specaugment, I’m really curious to know the result when applying fixed masking let’s say, mask_width_range=[0, 40] if you have some experiments on that?
Thank you.

**Comment by sw005320** [comment]
@pyf98, can you answer it?

**Comment by pyf98** [comment]
Hi @asadolahzade, thanks for your questions.

1) The BPE size is a hyper-parameter which affects performance and thus needs tuning. We have shown that both sizes work for this dataset, but they may not be optimal. You can try different values and select the best in your setting.

2) We do not have a direct comparison for the two cases. Again, it requires tuning. You can explore different hyper-parameters.
